base_classes/NXtransformations.nxdl.xml

```diff
@@ -108,6 +126,143 @@
 		* depends_on
 
 		as needed.
+
+		**Example 1: goniometer**
+
+		Position a sample mounted on a goniometer in the :ref:`McSTAS coordinate frame&lt;Design-CoordinateSystem&gt;`.
+
+		The sample is oriented as follows
+
+		.. math:: X_\text{lab} = R(\vec{v}_\omega, \omega) . 
+								 T(\vec{v}_z, \text{sam}_z) . 
+								 T(\vec{v}_y, \text{sam}_y) . 
+								 T(\vec{v}_x, \text{sam}_x) . 
+								 R(\vec{v}_\chi, \chi) . 
+								 R(\vec{v}_\varphi, \varphi) . X_s
+
+		where
+
+		* :math:`R(\vec{v},a)` is a rotation around vector :math:`\vec{v}` with angle :math:`a`
+		* :math:`T(\vec{u},t)` is a translation along vector :math:`\vec{u}` over a distance :math:`t`
+		* :math:`X_s` a coordinate in the sample reference frame.
+
+		.. code-block::
+
+		  entry:NXentry
+		    sample:NXsample
+		      depends_on=transformations/phi
+		      transformations:NXtransformations
+		        phi=0
+		          @depends_on=chi
+		          @transformation_type=rotation
+		          @vector=[-1 -0.0037 -0.002]
+		          @units=degrees
+		        chi=0
+		          @depends_on=sam_x
+		          @transformation_type=rotation
+		          @vector=[0.0046 0.0372 0.9993]
+		          @units=degrees
+		        sam_x=0
+		          @depends_on=sam_y
+		          @transformation_type=translation
+		          @vector=[1 0 0]
+		          @units=mm
+		        sam_y=0
+		          @depends_on=sam_z
+		          @transformation_type=translation
+		          @vector=[0 1 0]
+		          @units=mm
+		        sam_z=0
+		          @depends_on=omega
+		          @transformation_type=translation
+		          @vector=[0 0 1]
+		          @units=mm
+		        omega=174
+		          @depends_on=.
+		          @transformation_type=rotation
+		          @vector=[-1 0 0]
+		          @units=degrees
+
+		**Example 2: different coordinate system**
+
+		Define a laboratory reference frame with the X-axis along the beam and the Z-axis opposite to the direction of gravity.
```

Comment:
Here should be a note that NXcoordinate_system is another possibility to define such a "laboratory reference frame"

Reply:
I do not like at all that we never explicitly tell whether the here always Cartesian ref frames are considered always right or left handed ones surplus which direction of rotation we consider as positive clock/anti-clockwise

Reply:
X-axis along the beam is not specific enough but +X points in parallel and into the same direction as the propagating beam because otherwise the sign remains unclear or just implicitly defined, same so the z-axis might be parallel to gravity but which sign along z is positive in the direction of gravity or vice versa?